README.md

```diff
@@ -40,131 +53,177 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
+mounting it should be simple.
+To use this custom configuration class, the environment variables `DJANGO_SETTINGS_MODULE=imagetagger.settings_local`
+and `DJANGO_CONFIGURATION=MyCustomClass` must be set.
+
+If downloading zip files of Imagesets is desired, the feature can be enabled by settings `IT_ENABLE_ZIP_DOWNLOAD` to `true`. 
+A separate zip generation daemon must then be started via the following command.
+This feature is enabled and running automatically in Docker based deployments.
+```shell
+export DJANGO_CONFIGURATION=Prod
+./manage.py runzipdaemon
+```
 
-Example Nginx Config:
 
-```
-server {
-        listen 443;
-        server_name imagetagger.bit-bots.de;
-
-        ssl_certificate /etc/letsencrypt/certs/imagetagger.bit-bots.de/fullchain.pem;
-        ssl_certificate_key /etc/letsencrypt/certs/imagetagger.bit-bots.de/privkey.pem;
-        include /etc/nginx/ssl.conf;
-        include /etc/nginx/acme.conf;
-        ssl on;
-
-        client_max_body_size 4G;
-
-        access_log /var/log/nginx/imagetagger.bit-bots.de.access.log;
-        error_log /var/log/nginx/imagetagger.bit-bots.de.error.log;
-
-        location /static {
-                expires 1h;
-                alias /var/www/imagetagger;
-        }
-
-        location /ngx_static_dn/ {
-                internal;
-                alias /srv/data/imagetagger/storage/pictures/;
-        }
-
-        location / {
-                include uwsgi_params;
-                uwsgi_pass 127.0.0.1:4819;
-                uwsgi_read_timeout 120;
-        }
-}
-```
+### Minimal production Configuration
 
-If you want to provide zip files of image sets, set `ENABLE_ZIP_DOWNLOAD = True` in your `settings.py`.
-A daemon that creates and updates the zip files is necessary, you can start it with `./manage.py runzipdaemon`.
-Please take into account that the presence of zip files will double your storage requirement.
+In production, the following configuration values **must** be defined (as environment variables)
 
-Zip archive download via a script is also possible. The URL is `/images/imageset/<id>/download/`. A successful request
-returns HTTP 200 OK and the zip file. When the file generation is still in progress, HTTP 202 ACCEPTED is returned.
-For an empty image set, HTTP 204 NO CONTENT is returned instead of an empty zip archive.
+| Key | Description
+|---|---
+| `IT_SECRET_KEY` | The [django secret key](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-SECRET_KEY) used by ImageTagger. It is used for password hashing and other cryptographic operations.
+| `IT_ALLOWED_HOSTS` | [django ALLOWED_HOSTS](https://docs.djangoproject.com/en/3.0/ref/settings/#allowed-hosts) as comma separated list of values. It defines the hostnames which this application will allow to be used under.
+| `IT_DB_HOST` | Hostname (or IP-address) of the postgresql server. When deploying on kubernetes, the provided Kustomization sets this to reference the database deployment.
```

Comment:
I would also add `IT_DB_PASSWORD` here (see below) and possibly also mention user and db name settings and say that they default to `imagetagger`.